queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
```

Comment:
- [ ] Kindly add a query that can find all animals with a weight between 10.4kg and 17.3kg, as this is a requirement

All your queries
![image](https://user-images.githubusercontent.com/81932478/235502208-4a1795ec-1f3a-46b6-93b1-a66ea7d3c4c5.png)